src/Viewer/components/Monaco/utils.ts

```diff
@@ -60,6 +59,19 @@ const initMonacoEditor = (
         }
     );
 
+    let resizeTimeout: NodeJS.Timeout | null = null;
+    const resizeObserver = new ResizeObserver(() => {
+        if (null !== resizeTimeout) {
+            console.log("canceled");
```

Comment:
```suggestion
```
Assuming resizing the viewer layout is a common action, we might not want to log here.